test/Integration/Dialect/TOSA/bf16xbf16_sub_elem_2d_broadcast_scalar/bf16xbf16_sub_elem_2d_broadcast_scalar.mlir

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Could you please upload the dut.cc file for a reference?

Reply:
This is a XFAIL test, so no dut.cc at the moment. Will add the dut.cc when this test is fixed.